cli/cmd/sudo/user/remove.go

```diff
@@ -0,0 +1,53 @@
+package user
+
+import (
+	"errors"
+	"fmt"
+	"net/mail"
+
+	"github.com/rilldata/rill/cli/pkg/cmdutil"
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
+	"github.com/spf13/cobra"
+)
+
+func RemoveCmd(ch *cmdutil.Helper) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "remove <email>",
+		Short: "Remove a user",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+			client, err := ch.Client()
+			if err != nil {
+				return err
+			}
+
+			email := args[0]
+			if email == "" {
+				return errors.New("email is required")
+			}
+
+			if _, err := mail.ParseAddress(email); err != nil {
+				return fmt.Errorf("invalid email: %w", err)
+			}
+
+			orgs, err := client.ListOrganizationsByUser(ctx, &adminv1.ListOrganizationsByUserRequest{
+				Email: email,
+			})
+			if err != nil {
+				return fmt.Errorf("failed to list organizations for user %q: %w", email, err)
+			}
+
+			for _, org := range orgs.Organizations {
+				_, err = client.DeleteUser(ctx, &adminv1.DeleteUserRequest{Email: email, Organization: org.Id})
+				if err != nil {
+					return fmt.Errorf("failed to remove user %q from organization %q: %w", email, org.Id, err)
+				}
+			}
```

Comment:
Users exist separate of orgs, not nested under an org (check the database schemas) – so they can only be deleted once. Look at my previous comment, the user's org memberships probably need to be inspected and handled inside the `DeleteUser` RPC implementation.

To clarify, removing a user's _membership_ of an org is already possible using `rill user remove`. The purpose of this PR is to physically delete a user from our system (i.e. remove them from the `users` table and all references to them).

On reflection, to avoid ambiguity with `rill user remove`, let's call this command e.g. `rill user delete` (or can you think of a better name?)

Reply:
Oh! Ok that makes so much more sense. I'll leverage the schemas